### PR TITLE
allow everyone to see public documents #93

### PR DESCRIPTION
**Pull request by RONNCC**

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def initialize(user)
     end
 
     can :read, Document do |d|
-      d.public? and ( d.organization.blank? or d.organization.participants.include?(user.participant) )
+      d.public? or ( d.organization.participants.include?(user.participant) )
```
Comment (purelogiq) [inline app/models/ability.rb]: Crashes when there is a document that doesn't have an organization as specified in issue #93. (d.organization is nil and thus everything after is a null pointer exception).

Please modify this to use an if statement that checks whether it the doc has an org attached to it.

**Comment by purelogiq** [comment]
Also please make the same modification to the booth chair permissions in the same file. Booth Chairs are just participants for this purpose so the permission check should be the same.

**Comment by RONNCC** [comment]
@purelogiq should be fixed

**Comment by purelogiq** [comment]
Your change fixed the crashing.

Problem left is that booth chairs are still allowed to see private docs with no org.
Because booth chair permissions cascade with participant permissions you can literally just delete the booth chair Documents permission block and it will treat booth chairs as regular participants as the issue page specifies.

**Comment by RONNCC** [comment]
fixed

**Comment by purelogiq** [comment]
Looks good, doing final check.

**Pull request merged**